roles/ohpc_add_rabbitmq_agents/tasks/main.yaml

```diff
@@ -1,41 +1,44 @@
 ---
 - name: Clone the git repo to the machine
   git: 
-    repo: https://github.com/uabrc/rabbitmq_agents.git
-    dest: /opt/rabbitmq_agents
+    repo: "{{ rabbitmq_agents_repo }}"
+    dest: "{{ rabbitmq_agents_loc }}"
     version: develop
 
 - name: Make a copy of rabbit config example file
```

Comment:
Likely should adjust the name to not imply copy but rather "Install rabbitmp agents config file".

Reply:
fixed in 81f3658